### ISSUE TITLE
Mark of Quickening and Prismatic Vision

### ISSUE THREAD

**Issue by DireAvenger20** [SotMSteamMods/CauldronMods#159]
**Describe the bug**
With Mark of Quickening on Malichae to give him an extra play, skipping the play of djin card from Prismatic Vision causes the extra play to be lost.

**What does it affect**
Card(s): Mark of Quickening, Prismatic Vision
Deck(s): Malichae, The Stranger

**To Reproduce**
1. Play Mark of Quickening onto Malichae.
2. on Malichae's turn play Prismatic Vision.
3. After resolving the revealing and hand/trash parts of Prismatic Vision, skip the optional play of a Djin card.
4. This ends the play phase even though you would expect another play.

**Expected behavior**
After skipping the optional Prismatic Vision Djin card play, I would expect to be able to play another card.

**Comment by LNBertolini** [comment]
When you skip any card play, you skip all available card plays from that point onward - this also affects base Sentinels cards like Void Guard Writhe's Somber Tinker or Unity's Inspired Repair, which can matter under specific circumstances.

**Comment by Vilnor** [comment]
I don't think this is directly fixable, but I did find a discrepancy with the implementation of this card to base cards that grant extra actions.  No clue if this change will have any effect, but it's something.
Malichae's Unchackled, Doc Havoc's Reckless Charge have the same defect.
